nodes/celestia-node-troubleshooting.md

```diff
@@ -113,6 +140,56 @@ To show the keys you should add `--keyring-dir` like this example:
     --keyring-dir /home/user/celestia-<node-type>-location/keys/
 ```
 
+### Examples
+
+#### Mainnet Beta full and Mocha light
+
+This example uses a Mainnet Beta full node and a Mocha light node. When
+making the request:
+
+```bash
+❯ celestia blob get 1318129 0x42690c204d39600fddd3 0MFhYKQUi2BU+U1jxPzG7QY2BVV1lb3kiU+zAK7nUiY=
+{
+ "result": "RPC client error: sendRequest failed: http status 401 Unauthorized unmarshaling response: EOF"
+}
+```
+
+The request will go to the Mainnet Beta node, and 401 will show in this node's
```

Comment:
Maybe a quick mention of the reason for the 401 since the question you asked about why it failed is reasonable. "The request will go to the Mainnet Beta node, and a 401 will show in this node's logs. Note that a 401 is expected because the this namespace and blob was posted to Mocha and does not exist on Mainnet." Or something like that..